cardano-api/src/Cardano/Api/IO/Compat.hs

```diff
@@ -29,15 +29,15 @@ import           System.FilePath (splitFileName, (<.>))
 import           System.IO (hClose, openTempFile)
 #endif
 
-writeFileWithOwnerPermissions
+writeLazyByteStringFileWithOwnerPermissions
```

Comment:
Usually I would say lets put a deprecation warning but I'm pretty sure we're the only ones calling this :+1:

Reply:
Done!